exchange/auction_test.go

```diff
@@ -51,7 +53,24 @@ func TestCacheJSON(t *testing.T) {
 				t.Fatalf("Failed to load contents of file %s: %v", fileDisplayName, err)
 			}
 
-			runCacheSpec(t, fileDisplayName, specData)
+			runCacheSpec(t, fileDisplayName, specData, true, false)
+		}
+	}
+}
+
+// TestCacheJSON executes tests for all the *.json files in customcachekeytest.
+// customcachekey.json test here verifies custom cache key is used for vast video
+func TestCustomCacheKeyJSON(t *testing.T) {
+	if specFiles, err := ioutil.ReadDir("./customcachekeytest"); err == nil {
```

Comment:
Again, I think we should fail the test if we are unable to read the files in the specified dir for some reason and err is not nil.